main.js

```diff
@@ -0,0 +1,85 @@
+// the game board to display with the 1st peg being the "start" peg
+let board = [
+  [5,4,3,2,1],
+  [],
+  []
+]
+
+// function to execute to make the board visible
+function displayBoard() {
+  console.clear()
+  board.map(function(pegs) {
```

Comment:
wrong use of map, map returns something that you are not really doing anything with, this is less performance. The correct Array function is forEach for this use case.